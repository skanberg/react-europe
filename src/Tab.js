import React from "react";
import { Block } from "jsxstyle";

const Tab = ({ day, selectDay }) => (
  <Block
    width="20%"
    textAlign="center"
    paddingTop={10}
    paddingBottom={10}
    borderBottom="1px solid #2F4592"
  >
    <div
      onClick={() => {
        selectDay(day);
      }}
    >{`Day ${day}`}</div>
  </Block>
);

export default Tab;
