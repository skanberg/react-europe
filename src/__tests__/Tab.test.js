import React from "react";
import renderer from "react-test-renderer";
import Tab from "../Tab";

test("renders with active tab", () => {
  const tree = renderer.create(<Tab day={1} selected/>).toJSON();
  expect(tree).toMatchSnapshot();
});

test("renders with inactive tab", () => {
  const tree = renderer.create(<Tab day={1} selected={false} />).toJSON();
  expect(tree).toMatchSnapshot();
});
