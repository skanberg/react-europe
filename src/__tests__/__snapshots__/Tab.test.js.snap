exports[`test renders with active tab 1`] = `
<div
  className="w-20P text-align-center pt-15 pb-15 _1086erg fw-700">
  <div
    onClick={[Function]}>
    Day 1
  </div>
</div>
`;

exports[`test renders with inactive tab 1`] = `
<div
  className="w-20P text-align-center pt-15 pb-15 _1086erg fw-400">
  <div
    onClick={[Function]}>
    Day 1
  </div>
</div>
`;
